csharp/src/Microsoft.ML.OnnxRuntime/Microsoft.ML.OnnxRuntime.csproj

```diff
@@ -1,7 +1,7 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netstandard1.1</TargetFramework>
+    <TargetFrameworks>netstandard1.1;netstandard2.0</TargetFrameworks>
```

Comment:
Does this change increase the size of the binary produced?

Reply:
Afaik no. I didn't look deeply into the packages generated by our build process here before and after, but in general adding a new target does not change the existing target's binary but produces an _additional_ binary, that is included in the nuget package. See e.g. \[0\] for an example (screenshot of the latest version of the `Netwonsoft.Json` package opened in [NuGetPackageExplorer](https://github.com/NuGetPackageExplorer/NuGetPackageExplorer)). When one adds the reference to their project, the SDK decides during the build which version suits best and just uses that one binary. Specifically, when adding a ns2.0 target, all consuming projects that support ns2.0 will use that ns2.0 binary by default, while consumers that do support ns1.1 but not ns2.0 will use the older target. Thus, this might increase the NuGet package download size a tiny bit (the managed lib is just 115kb before compression in the package), for the advantage of the greatly simplified dependency graph, but afaik multi targeting does not increase the deployed binaries' sizes.

\[0\]:
![image](https://user-images.githubusercontent.com/5229609/128150524-da3bb19d-d8ff-4059-8ffb-8fa12f1f4288.png)